dynamics-nav/DateTime-Functions.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "DateTime Functions"
+description: "This article references the DateTime functions in Microsoft Dynamics NAV."
```

Comment:
Same as comments above and for below, if you want to capitalize Functions/Functionality/etc. you can to keep consistency with the others--just a suggestion!